test/fixtures/mson-mixin-nonexistent.json

```diff
@@ -0,0 +1,109 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "<API name>",
+  "description": "<API description>\n\n",
+  "element": "category",
+  "resourceGroups": [],
```

Comment:
Refract Elements are expanded in place of original "DataStructure". 
One of acceptance criteria is be backward compatible.

But in general it ca be anywhere